js_modules/dagit/packages/core/src/assets/__fixtures__/AssetTables.fixtures.ts

```diff
@@ -309,6 +309,97 @@ export const SingleAssetQueryLastRunFailed: MockedResponse<SingleAssetQuery> = {
   },
 };
 
+export const AssetCatalogTableMockAssets: Extract<
+  AssetCatalogTableQuery['assetsOrError'],
+  {__typename: 'AssetConnection'}
+>['nodes'] = [
+  {
+    id: '["dashboards", "cost_dashboard"]',
+    __typename: 'Asset',
+    key: {
+      path: ['dashboards', 'cost_dashboard'],
+      __typename: 'AssetKey',
+    },
+    definition: null,
+  },
+  {
+    id: '["dashboards", "traffic_dashboard"]',
+    __typename: 'Asset',
+    key: {
+      path: ['dashboards', 'traffic_dashboard'],
+      __typename: 'AssetKey',
+    },
+    definition: null,
+  },
+  {
+    id: 'test.py.repo.["good_asset"]',
+    __typename: 'Asset',
+    key: {
+      path: ['good_asset'],
+      __typename: 'AssetKey',
+    },
+    definition: buildAssetNode({
+      id: 'test.py.repo.["good_asset"]',
+      groupName: 'GROUP2',
+      partitionDefinition: null,
+      hasMaterializePermission: true,
+      description:
+        'This is a super long description that could involve some level of SQL and is just generally very long',
+      repository,
+      __typename: 'AssetNode',
+    }),
+  },
+  {
+    id: 'test.py.repo.["late_asset"]',
+    __typename: 'Asset',
```

Comment:
You could probably use `buildAsset` for these, then you won't need the `__typename`

Reply:
Oh good call I'll upgrade these while I'm here

Reply:
in vscode if you use regex find:
```
\s\{[\s\r]*(__typename:\s*'([^']*)')(,)?
```
replace:
```
 build$2({
```